docs/getting-started/configuring-your-repository.md

```diff
@@ -0,0 +1,47 @@
+# Configuring your repository
+
+<!--TODO
+    - Review the intro and consider integrating the remaining info gathered from previous versions
+
+We recommend that you spend some time reviewing the Codacy analysis settings to focus on the tools and code patterns that bring more value to your team. Importantly, this eliminates the risk of having false positives block the work of your team once you start using Codacy as a quality gate.
+-->
+
+Once you've [added your first repository](codacy-quickstart.md), it's important that you configure the analysis tools to match the use cases of your team and avoid time-consuming false positives. In particular, you should configure any coding conventions and best practices that your team may already be following or that you want to enforce.
```

Comment:
What if we mentioned the false positives only at the end of this paragraph?

That way we could have a more straightforward sentence flow because we would mention everything related to matching the use cases of the team (coding conventions, best practices, and quality standards to enforce), and only then mention the importance of avoiding false positives. I feel that like this, the paragraph is switching back and forth between the two things.

Reply:
OK, I tried reversing the sentence order